### PR TITLE
General Updates and fixes for documentation

### PR DESCRIPTION
**Pull request by josh-termly**

### CODE REVIEW

**Review by ghost — APPROVED** [review]
lgtm

File: endpoints/delete_collaborators.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The response is an array of the following objects. If the deletion was successfu
 
 If there is an error when deleting, the objects have the following shape
 
-```json
+```
```
Comment (dkreft-termly) [inline endpoints/delete_collaborators.md:32]: Your intending here appears to be about 4 spaces per level...compare that with `post_cookies.md`...consistency is good, and I'd lean strongly towards the 2-space indents that I've seen in all your previous PRs.

File: endpoints/post_cookies.md [hunk 2]
```diff
@@ -34,14 +33,14 @@ Creates cookies. The requst body has the following shape:
 - `de` is the description in German
 - `en_uk` is the description in UK English
 - `en_us` is the description in US English
-- `expire` is the expiry of the cookie
+- `expire` is the expiry of the cookie in seconds
```
Comment (dkreft-termly) [inline endpoints/post_cookies.md:36]: This should probably be `expires_at`, `expires_on` or `expiry`....it's kinda weird having a verb as a key.

File: endpoints/post_cookies.md [hunk 3]
```diff
@@ -141,7 +144,7 @@ Creates multiple cookies for multiple websites.
 
 POST `/api/v3/websites/cookies`
 
-## Query
+## Request Body
```
Comment (dkreft-termly) [inline endpoints/post_cookies.md:147]: In the sample body, below, can you just leave out the null key-value pairs? This seems to be communicating that we have to supply *all* pairs, even if there is no value...which seems awfully draconian.

File: endpoints/put_collaborators.md [hunk 4]
```diff
@@ -47,7 +47,7 @@ The response is an array of objects. If the collaborator can be updated successf
 * `email` is the email address of the collaborator
 * `first_name` is the Given name of the collaborator if provided by the collaborator upon signing in for the first time.
 * `last_name` is the Surname of the collaborator if provided by the collaborator upon signing in for the first time.
-* `invitation_url` is the URL sent to the collaborator so they can activate their account
+* `invitation_url` is the URL sent to the collaborator so they can activate their account.
```
Comment (dkreft-termly) [inline endpoints/put_collaborators.md:50]: "they" is a 2nd-person plural pronoun, whereas "collaborator" is singular.

Yes, I'm an old-skool grammar nazi.

You can more concisely (and properly) state this as:

"is the account activation URL"

File: endpoints/put_cookies.md [hunk 5]
```diff
@@ -29,13 +28,14 @@ Updates cookies. The request body has the following shape:
...
 - `account_id` is the unique identifier of the account
+- `id` is the unique identifier of the cookie
 - `category` is the cookie category
 - `country` is the country that sourced the cookie, based on a geoip lookup
 - `domain` is the domain of the cookie
 - `de` is the description in German
 - `en_uk` is the description in UK English
 - `en_us` is the description in US English
-- `expire` is the expiry of the cookie
+- `expire` is the expiry of the cookie in seconds
```
Comment (dkreft-termly) [inline endpoints/put_cookies.md:38]: See previous comment about the name of this field.

**Pull request merged**